x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/total_stats.tsx

```diff
@@ -6,32 +6,34 @@
 
 import React from 'react';
 import { useValues } from 'kea';
-import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiStat } from '@elastic/eui';
 
 import { TOTAL_QUERIES, TOTAL_DOCUMENTS, TOTAL_CLICKS } from '../../analytics/constants';
+import { AnalyticsCards } from '../../analytics';
 
 import { EngineOverviewLogic } from '../';
 
 export const TotalStats: React.FC = () => {
   const { totalQueries, documentCount, totalClicks } = useValues(EngineOverviewLogic);
 
   return (
-    <EuiFlexGroup>
-      <EuiFlexItem>
-        <EuiPanel data-test-subj="TotalQueriesCard">
-          <EuiStat title={totalQueries} description={TOTAL_QUERIES} titleColor="primary" />
-        </EuiPanel>
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiPanel data-test-subj="TotalDocumentsCard">
-          <EuiStat title={documentCount} description={TOTAL_DOCUMENTS} titleColor="primary" />
-        </EuiPanel>
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiPanel data-test-subj="TotalClicksCard">
-          <EuiStat title={totalClicks} description={TOTAL_CLICKS} titleColor="primary" />
-        </EuiPanel>
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <AnalyticsCards
+      stats={[
```

Comment:
Haha I just realized turning this into a reusable component actually increased the total line count by 2 😭 Oh well, it was maybe a bit overkill for just this 1 scenario, but I figured if we ever need to change the styling across the board for all of the Analytics stat cards (potentially the titleColor I imagine) it becomes much quicker and easier to do so in just 1 place.